test/evergreen.yml

```diff
@@ -2971,14 +3017,21 @@ buildvariants:
       CFLAGS="-fsanitize=undefined -fno-omit-frame-pointer -fno-optimize-sibling-calls -O1 -ggdb"
       CXXFLAGS="-fsanitize=undefined -fno-omit-frame-pointer -fno-optimize-sibling-calls -O1 -ggdb"
     posix_configure_flags:
-      --enable-silent-rules --enable-strict --enable-diagnostic
-      --disable-static --prefix=$(pwd)/LOCAL_INSTALL
+      --enable-silent-rules
+      --enable-strict
+      --enable-diagnostic
+      --disable-static
+      --enable-tcmalloc
+      --prefix=$(pwd)/LOCAL_INSTALL
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
     make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
     test_env_vars:
       UBSAN_OPTIONS="detect_leaks=1:disable_coredump=0:external_symbolizer_path=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer:halt_on_error=1:print_stacktrace=1"
-      LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so LD_LIBRARY_PATH=$(pwd)/.libs
-      PATH=/opt/mongodbtoolchain/v3/bin:$PATH top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+      LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libeatmydata.so
+      LD_LIBRARY_PATH=$(pwd)/.libs:$(git rev-parse --show-toplevel)/TCMALLOC_LIB/lib
```

Comment:
It looks like `$git rev-parse --show-toplevel)` tries to get the top src directory - same value as the following `top_srcdir` expansion? Not sure if there's a way to avoid duplication here (e.g. using an expansion to denote another expansion).

Reply:
I have used `top_srcdir` for this.